src/Microsoft.Dnx.Runtime/DependencyManagement/LockFilePackageLibrary.cs

```diff
@@ -17,5 +17,13 @@ public class LockFilePackageLibrary
         public string Sha512 { get; set; }
 
         public IList<string> Files { get; set; } = new List<string>();
+
+        public LockFilePackageLibrary() { }
+
+        public LockFilePackageLibrary(string name, SemanticVersion version)
```

Comment:
Nam and SemanticVersion already have setters.. This ctor is weird - it is public API and gives an impression that these two properties are somehow special. I have a feeling that we are polluting public API without a good reason - without the ctor the class is as testable as it was before.